thrift_tools/printer.py

```diff
@@ -36,7 +36,8 @@ def print_color(s, color_id, output=sys.stdout):
     output.flush()
 
 
-def print_msg(timestamp, src, dst, msg, format_opts,
+def print_msg(timestamp, src, dst, msg, 
+              ,
```

Comment:
this seems unintended

Reply:
yes, it should not to be committed, it already fixed.